website/build.gradle

```diff
@@ -104,3 +106,32 @@ task testWebsite(type: Exec) {
 task preCommit {
   dependsOn testWebsite
 }
+
+task mergeWebsite << {
+  /* Disabled for now, for testing
+  exec {
+    executable 'git'
```

Comment:
FYI, when I was previously working on the design, I looked into other ASF projects, and this is a common pattern. I believe some projects were using a Groovy helper library for git interactions. If we find invoking git manually to be tedious / flaky, we can look further into the helper library.

Reply:
Added the org.ajoberstar.git-publish and org.ajoberstar.grgit plugins